### PR TITLE
feat: Show compute units consumed

### PR DESCRIPTION
**Pull request by Arrowana**
Plugged to right RPC node we can see some data

### CODE REVIEW

File: src/pages/TransactionDetailsPage.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export function TransactionDetailsPage({ signature: raw }: SignatureProps) {
     if (decoded.length === 64) {
       signature = raw;
     }
-  } catch (err) {}
+  } catch (err) { }
```
Comment (joncinque) [inline src/pages/TransactionDetailsPage.tsx]: It looks like prettier hasn't been run on this recently, so I'll give you a pass on this and the other whitespace change

File: src/pages/TransactionDetailsPage.tsx [hunk 2]
```diff
@@ -332,6 +332,15 @@ function StatusCard({
...
         )}
 
+        {computeUnitsConsumed && (
+          <tr>
+            <td>Compute units consumed</td>
+            <td className="text-lg-end">
+              {computeUnitsConsumed.toLocaleString("en-US")}
+            </td>
+          </tr>
+        )}
```
Comment (joncinque) [inline src/pages/TransactionDetailsPage.tsx]: For a system instruction, in which this is `0`, you get a funny result :sweat_smile: can you do?
```suggestion
        {computeUnitsConsumed !== undefined && (
          <tr>
            <td>Compute units consumed</td>
            <td className="text-lg-end">
              {computeUnitsConsumed.toLocaleString("en-US")}
            </td>
          </tr>
        )}
```

Comment (joncinque) [reply src/pages/TransactionDetailsPage.tsx]: image

Comment (Arrowana) [reply src/pages/TransactionDetailsPage.tsx]: The classic react noob bug, fixed

File: src/pages/TransactionDetailsPage.tsx (hunk 1 above)
Comment (Arrowana) [reply src/pages/TransactionDetailsPage.tsx]: I did not notice, i reverted the linter auto fixes

**Review by nickfrosty — APPROVED** [review]

**Review by joncinque — APPROVED** [review]
Looks good to me! Now I just need to get privileges to merge and deploy...

**Comment by ngundotra** [comment]
@joncinque you should now have admin privileges!

**Pull request merged**